### PR TITLE
Machine Learning

### PR DESCRIPTION
**Pull request by Deepanshu0810**
### Description
In this PR I have added a new section Machine Learning in this resource

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 - [Git and GitHub](#git-and-github)
 - [Good first issue finder](#good-first-issue-finder)
 - [Web3](#web3)
+- [Machine Learning](#machine-learning)
```
Comment (Ashish-khanagwal) [inline README.md:70]: Hey, please add `Machine learning` in alphabetical order

File: README.md [hunk 2]
```diff
@@ -254,4 +255,25 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 - :books: [Best roadmap](https://vitto.cc/web3-and-solidity-smart-contracts-development-roadmap/#d12f)
 - :video_camera: [Web3 Roadmap](https://www.youtube.com/watch?v=Lz3m4BI1EfY&t=5s)
 
+---
+
+## Machine Learning
```
Comment (Ashish-khanagwal) [inline README.md:260]: Add this section in alphabetical order too

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Deepanshu0810) [reply README.md:70]: Ok,
Thank you for the review

**Comment by Deepanshu0810** [comment]
> 

Thank You, for providing such awesome opportunities, looking forward to contribute as much as I can for the community

**Pull request merged**